clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -2959,8 +2959,8 @@ static bool ParseFrontendArgs(FrontendOptions &Opts, ArgList &Args,
   if (Args.hasArg(OPT_clangir_verify_diagnostics))
     Opts.ClangIRVerifyDiags = true;
 
-  if (Args.hasArg(OPT_fclangir_call_conv_lowering))
-    Opts.ClangIREnableCallConvLowering = true;
+  if (Args.hasArg(OPT_fclangir_analysis_only))
+    Opts.ClangIRAnalysisOnly = true;
```

Comment:
`ClangIREnableCallConvLowering` is orthogonal to what you are changing here (looks like you just need to add the extra logic but not delete anything), was this a mistake?

Reply:
Oh, this should be a merge oversight. Fixed.